lib/unimatrix/authorization/version.rb

```diff
@@ -1,5 +1,5 @@
 module Unimatrix
   module Authorization
-    VERSION = "1.1.2"
+    VERSION = File.read( File.expand_path( '../../../../VERSION', __FILE__ ) ).chomp
```

Comment:
Yea we ran into that problem where it was giving us the Version of the app using the SDK and not the version of the SDK hence, this...